clang/test/CodeGenOpenCL/check-atomic-alignment.cl

```diff
@@ -1,8 +1,6 @@
 // NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
-// RUN: %clang_cc1 -O0 -cl-std=CL2.0 -triple amdgcn-amd-amdhsa -target-cpu gfx942 \
-// RUN:   %s -emit-llvm -o - | FileCheck %s
-
-// REQUIRES: amdgpu-registered-target
+// RUN: %clang_cc1 -O0 -cl-std=CL1.2 -triple amdgcn-amd-amdhsa -target-cpu gfx942 \
```

Comment:
I just realized you are testing with the flat atomic, so maybe this should stay as 2.0. I'm surprised it actually worked without flat address space